addons/dark-www/forums_preview.js

```diff
@@ -55,7 +55,7 @@ export default async function ({ addon, console }) {
             previewRoot = node;
           }
           if (node.tagName === "LINK" && node.href.endsWith("djangobb_forum/css/pygments.css")) {
-            preview.contentDocument.head.appendChild(createStyle(addon.self.dir + "/experimental_scratchr2.css"));
+            preview.contentDocument.head.appendChild(createStyle(addon.self.dir + "/scratchr2.css"));
             preview.contentDocument.head.appendChild(createStyle(addon.self.dir + "/experimental_forums.css"));
```

Comment:
?

Reply:
I think that file didn't actually exist and another PR removed that line of the js file (causing the conflict)